src/pages/norme-e-riferimenti/linee-guida-di-design/requisito-4-3/azione-1.yaml

```diff
@@ -0,0 +1,536 @@
+        addonStyle: btn-share-hero text-primary mb-5 mt-lg-4 ms-auto
+        icon:
+          icon: sprites.svg#it-more-items
+          size: sm
+          color: primary
+          addonClasses: ms-3
+      list:
+        isShare: true
+
+    # KANGAROO: METADATA
+    kangaroo:
+      id: metadati-e-link
+      titleSr: Metadati e link per approfondire
+      navposition:
+        items:
+          - title: Ti trovi in
+            icon:
+              icon: "sprites.svg#it-folder"
+              size: "sm"
+              align: "middle"
+              color: "secondary"
+            label: Linee guida di design #C
+            url: "../../" #M
+          - title: Nel requisito #C
+            icon:
+              icon: "sprites.svg#it-file"
+              size: "sm"
+              align: "middle"
+              color: "secondary"
+            label: "4.3. Semplicità di consultazione ed esperienza d'uso" #C
+            url: "../" #M
+      # otherInfo:
+      #   items:
+      #     - title: Rispettando
+      #       icon:
+      #         icon: "sprites.svg#it-pa"
+      #         size: "sm"
+      #         align: "middle"
+      #         color: "secondary"
+      #       label: Design standard X #C
+      #       url: "#" #M
+      # personalInfo:
+      #    items: #C
+      #     - title: Sei
+      #       icon:
+      #         icon: "sprites.svg#it-user"
+      #         size: "sm"
+      #         align: "middle"
+      #         color: "secondary"
+      #       label: Designer, Developer #C
+      #     - title: Vuoi
+      #       icon:
+      #         icon: "sprites.svg#it-flag"
+      #         size: "sm"
+      #         align: "middle"
+      #         color: "secondary"
+      #       label: Progettare, Realizzare #C
+      tagsLabel: Parliamo di
+      color: secondary
+      icon:
+        icon: sprites.svg#it-horn
+        size: sm
+        color: secondary
+      chips: #C
+        - label: Normativa
+          url: "#"
+          color: primary
+          size: lg
+        - label: Cultura progettuale
+          url: "#"
+          color: primary
+          size: lg
+        - label: Progettazione servizi
+          url: "#"
+          color: primary
+          size: lg
+
+  # ///////////////////
+  # SECTION EDITORIAL
+  # ///////////////////
+  sectionsEditorial :
+
+
+    - title: Scopri le indicazioni a supporto di questa azione nel Manuale operativo di design #C
+      menu: false
+      centered: true
+      background: null
+      # full: true
+      # text: | #C
+      # Lorem **ipsum**
+      components:
+
+        # TXT BLOCK
+        - name: TextImageCta
+          title: L'approccio progettuale #C
+          headingLevel: 4
+          text: | #C
+            Scegli l’approccio progettuale tra quelli proposti per affrontare al meglio il processo sotto diversi punti di vista: quello delle persone, delle procedure e della tecnologia.
+          ctas:
+          - label: Vai all'argomento sul Manuale #C
```

Comment:
Non possiamo usare la formula "Vai all'argomento sul Manuale" su tutte le CTA diverse in pagina. I contenuti dei link devono essere indicativi di cosa stai cliccando per l'accessibilità. Possiamo anche ottenerlo con testo nascosto in più che ripete il titolo del paragrafo, ma preferirei evitare se possibile e trovare formule più chiare possibili. Parliamone.

Reply:
@danielaiozzo